serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -335,7 +335,11 @@ export class _Exporter {
             localCanvas.id = "WriteCanvas";
             localCanvas.width = 2048;
             localCanvas.height = 2048;
-            this._localEngine = new Engine(localCanvas, true, { premultipliedAlpha: false, preserveDrawingBuffer: true });
+            if (Tools.IsSafari()) {

```

Comment:
@deltakosh, I noticed in my local build that I'm not able to "see" the new Tools utility method I added, causing the build to fail here. Is there anything I need to do to make sure that this method is exported properly?

Reply:
Well yeah I know, you need to build the .d.ts first and then recompile but no worries the CI will take care of it

Reply:
Looks like the CI is failing for the same reason:
https://dev.azure.com/babylonjs/ContinousIntegration/_build/results?buildId=3030&view=logs&j=182f86de-37f5-5af4-c789-e92a7c8fa3dc&t=14c2bff8-fe23-5e9c-07aa-9b09ac155f63&l=169

I'm not sure what I'm doing wrong here

Reply:
Are you sure this is the correct Tools (the one from bjs?)

Reply:
I made the change to src/Misc/Tools.ts...

Nevermind I'm just being silly. I accidentally put the change as a part of AsyncLoop